src/components/CountryDetails.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import countries from '../countries.json'
+import { Link } from 'react-router-dom'
+
+export default function CountryDetails(props) {
+    const country = countries.find(country => country.cca3 === props.match.params.id)
+    const capital = country.capital.map(cap => <p key={cap}>{cap}</p>)
```

Comment:
You only need to add a unique key for elements inside a list. So here the key is not needed.

This will also work as `country.capital`